mock/py/mockbuild/package_manager.py

```diff
@@ -338,3 +350,50 @@ def builddep(self, *pkgs, **kwargs):
                 if 'no such command: builddep' in line.lower():
                     raise BuildError("builddep command missing.\nPlease install package dnf-plugins-core.")
             raise
+
+
+# class for microdnf
+class MicroDnf(_PackageManager):
+    name = 'microdnf'
+
+    def __init__(self, config, buildroot, plugins, bootstrap_buildroot):
+        super(MicroDnf, self).__init__(config, buildroot, plugins, bootstrap_buildroot)
+        self.command = config['microdnf_command']
+        self.install_command = config['microdnf_install_command']
+        self.builddep_command = [self.command, 'builddep']
+        if bootstrap_buildroot is None:
+            self._check_command()
+        self.resolvedep_command = [self.command, 'repoquery', '--resolve', '--requires']
+
+    @traceLog()
+    def build_invocation(self, *args):
+        if 'microdnf_builddep_opts' not in self.config:
+            self.config['microdnf_builddep_opts'] = self.config['yum_builddep_opts']
+        if 'microdnf_common_opts' not in self.config:
+            self.config['microdnf_common_opts'] = self.config['yum_common_opts'] + \
+                                                  ['--disableplugin=local', '--setopt=deltarpm=False']
+        return super(MicroDnf, self).build_invocation(*args)
+
+    @traceLog()
+    def initialize_config(self):
+        if 'microdnf.conf' in self.config:
+            config_content = self.config['microdnf.conf']
+        else:
+            config_content = self.config['yum.conf']
+            config_content = self.replace_in_config(config_content)
+        check_yum_config(config_content, self.buildroot.root_log)
+        util.mkdirIfAbsent(self.buildroot.make_chroot_path('etc', 'microdnf'))
+        # Bug in microdnf , microdnf takes the repository resources from the same location as dnf
```

Comment:
Can you reference the bug?

Reply:
I was looking at microdnf code today, And I realized that I have made a mistake.